scripts/annotations_import.py

```diff
@@ -0,0 +1,174 @@
+    (['Alteration, removal, or addition of entire section(s) of a webpage'],
+     'alteration_entire_sections'),
+    (['Alteration, removal, or addition of an entire webpage or document'],
+     'alteration_entire_webpage_or_document'),
+    (['Overhaul, removal, or addition of an entire website'],
+     'alteration_entire_website'),
+    (['Alteration, removal, or addition of datasets'],
+     'alteration_dataset')]]
+
+STRING_ANNOTATION_ATTRIBUTES = [AnnotationAttributeInfo(*info) for info in [
+    (['Is this primarily a content or access change (or both)?'],
+     'content_or_access_change'),
+    (['Brief Description'],
+     'brief_description'),
+    (['Topic 1'],
+     'topic_1'),
+    (['Subtopic 1a'],
+     'subtopic_1a'),
+    (['Subtopic 1b'],
+     'subtopic_1b'),
+    (['Topic 2'],
+     'topic_2'),
+    (['Subtopic 2a'],
+     'subtopic_2a'),
+    (['Subtopic 2b'],
+     'subtopic_2b'),
+    (['Topic 3'],
+     'topic_3'),
+    (['Subtopic 3a'],
+     'subtopic_3a'),
+    (['Subtopic 3b'],
+     'subtopic_3b'),
+    (['Any keywords to monitor (e.g. for term analyses)?'],
+     'keywords_to_monitor'),
+    (['Further Notes'],
+     'further_notes'),
+    (['Ask/tell other working groups?'],
+     'ask_tell_other_working_groups'),
+
+    # Including this so that we can eventually map it to
+    # users in the database
+    (['Who Found This?'],
+     'annotation_author')]]
+
+def get_attribute_value(attribute_info, csv_row):
+    for column_name in attribute_info.column_names:
+        if column_name in csv_row:
+            return csv_row[column_name].strip()
+
+    # Despite being raised in a row-level function, this error means that the
+    # whole sheet is missing a column, so we don't catch and allow it to crash
+    raise CsvSchemaError(f'Expected to find one of {attribute_info.column_names} '
+                         f'in {csv_row.keys()}')
+
+def create_annotation(csv_row, is_important_changes):
+    annotation = {}
+
+    for attribute_info in BOOL_ANNOTATION_ATTRIBUTES:
+        attribute_value = get_attribute_value(attribute_info, csv_row)
+        annotation[attribute_info.json_key] = attribute_value == '1'
+    for attribute_info in STRING_ANNOTATION_ATTRIBUTES:
+        attribute_value = get_attribute_value(attribute_info, csv_row)
+        annotation[attribute_info.json_key] = attribute_value
+
+    # This will need additional logic to determine the actual sheet schema
+    annotation['annotation_schema'] = 'edgi_analyst_v2'
+
+    significance = 0.0
+    if is_important_changes:
+        importance_significance_mapping = {
+            'low': 0.5,
+            'medium': 0.75,
+            'high': 1.0
+        }
+        row_importance = csv_row['Importance?'].lower().strip()
+        significance = importance_significance_mapping.get(row_importance, 0.0)
+    annotation['significance'] = significance
+
+    return annotation
+
+def post_annotation(change_ids, annotation):
+    client = Client.from_env()
```

Comment:
Minor nit — might be nice to only create one client and pass it in instead of creating a new one for every row.

Or do we even need a function for this at that point, since the entire function body would just be:

```py
return client.add_annotation(**change_ids, annotation=annotation)
```

Reply:
:+1: